### ISSUE TITLE
No focus movement when activating the "information" button

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#895]
## Bug Report

**Problematic behavior**
When you are in the meeting, using keyboard navigation, when you reach the "information" button. Once the button is triggered, the focus does not move.

**Expected behavior/code**
When activating the "information" button with the keyboard, focus must move to the close button of the information panel.

**Additional context/Screenshots**
<img width="406" height="508" alt="Image" src="https://github.com/user-attachments/assets/043c9734-e068-4d92-949d-f416b5d57f88" />

**Comment by Ovgodd** [comment]
Hello @cyberbaloo  ! this will be fixed in this PR : https://github.com/suitenumerique/meet/pull/869
